src/ApparentHorizons/YlmSpherepack.hpp

```diff
@@ -0,0 +1,349 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+#include "ApparentHorizons/YlmSpherepackHelper.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+/// \ingroup Spectral
+/// \brief C++ interface to SPHEREPACK.
+class YlmSpherepack {
```

Comment:
If this is stored for long periods of time and may need to be check pointed then a `pup` function is necessary.

Reply:
The plan so far was to not pup YlmSpherepack, but instead in the pupper of Strahlkorper recreate the YlmSpherepack if unpacking.